core/templates/css/oppia.css

```diff
@@ -3278,6 +3280,11 @@ md-card.preview-conversation-skin-supplemental-card {
   margin-right: 5px;
   vertical-align: middle;
 }
+
+.fas {
+  margin-right: 5px
+}
```

Comment:
@nithinrdy Is this change okay?

Reply:
Hey @divyanshkhetan the code looks alright, could you please update the proof of changes section? Specifically the first image where the classroom icon seems broken and displays a link instead of the actual icon. Thanks!

Reply:
@nithinrdy Done. Please have a look if this is okay. Thanks!

Reply:
@nithinrdy can you please review this PR?